crates/matrix-sdk/src/client/mod.rs

```diff
@@ -474,7 +474,7 @@ impl Client {
     /// The server used by the client.
     ///
     /// See `Self::server` to learn more.
-    fn server(&self) -> Option<&Url> {
+    pub fn server(&self) -> Option<&Url> {
```

Comment:
Is it intended to make it `pub`? Looks like `pub(crate)` would have been sufficient, but I can see the appeal of making this public for users too.

Reply:
`pub(crate)` would be fine for this patch, but I believe it can be useful for someone else.